src/main/java/alfio/util/ImageUtil.java

```diff
@@ -40,43 +41,53 @@
 public final class ImageUtil {
     private ImageUtil() {
     }
-
-    public static byte[] createQRCode(String text) {
+    static int width = 200;
+    static int height = 200;
+    static int border = 0;
```

Comment:
`static int` should be `private static final int` for width, height and border, additionally, they should be in UPPERCASE